ui/src/utils/plugins.js

```diff
@@ -90,7 +90,7 @@ export const pollJobPlugin = {
 
           // Ensure we refresh on the same / parent page
           const currentPage = this.$router.currentRoute.path
-          const samePage = originalPage === currentPage || originalPage.startsWith(currentPage + '/')
+          const samePage = options.originalPage === currentPage || options.originalPage.startsWith(currentPage + '/')
```

Comment:
```suggestion
          const samePage = originalPage === currentPage || originalPage.startsWith(currentPage + '/')
```

My bad @utchoang

Reply:
@davidjumani If I don't set the originalPage it will be null, when I run the test, I get the following error. It seems that `options.originalPage = options.originalPage || this.$router.currentRoute.path` it doesn't override originalPage
![image](https://user-images.githubusercontent.com/13766648/127431402-68eab928-8d0d-4a49-b9ca-d5dbf352c6c9.png)

Reply:
originalPage is null, and can't startsWith by this

Reply:
What do you think about my suggestion?
`const originalPagePath = originalPage || this.$router.currentRoute.path`
and
`const samePage = originalPagePath === currentPage || originalPagePath.startsWith(currentPage + '/')`

Reply:
Got it! Thanks